src/components/Footer.css

```diff
@@ -0,0 +1,15 @@
+.App-footer {
+  display: flex;
+  justify-content: space-around;
+  border-top: 5px solid var(--color-icons);
+  background-color: var(--color-background);
+}
+
+.App-footer__navigation {
+  display: flex;
+  justify-content: space-evenly;
```

Comment:
because the div is only as wide as the content, this doesn't change anything and you can leave it out. a possibility would to add `width: 100%` to span the navigation across the whole width of it's parent element, the footer.

Reply:
Very helpful ! Thanks John !